rllib/evaluation/collectors/simple_list_collector.py

```diff
@@ -730,6 +730,7 @@ def get_inference_input_dict(self, policy_id: PolicyID) -> Dict[str, TensorType]
                 if data_col
                 in [
                     SampleBatch.OBS,
+                    SampleBatch.INFOS,
```

Comment:
In method `get_inference_input_dict()`, the key `INFOS` should be shift togather with `ODS` / `T`.

```python
# initial step
obs = env.reset()
info = {}
t = -1
state = policy.model.get_initial_state()

results = policy.compute_single_action(obs,        # obs  at `t = -1`
                                       info=info,  # info at `t = -1`
                                       state=state)
action, state, *_ = results

# interaction (t = 0)
obs, reward, done, info = env.step(action)
t = 0
results = policy.compute_single_action(obs,        # obs  at `t = 0`
                                       info=info,  # info at `t = 0`
                                       state=state)
action, state, *_ = results

# interaction (t = 1)
obs, reward, done, info = env.step(action)
t = 1
...
```